src/App.js

```diff
@@ -1,13 +1,33 @@
-import React from 'react';
+import React from "react";
```

Comment:
- Great job on the assignment Timi. 
- You are properly handling state and all the events in React. You split Your app into smaller components to make them modular. 
- Your todo app is fully functional and it is possible to add new tasks. The remove and mark complete functionality still needs a little work , but this is for today part of the project 
- Great effort doing search functionality 🥇 🎖 
- Your design looks really good, nice job with this one 👍
- Your git workflow is really fluid and detailed, keep up the good job with this.